pilot/pkg/config/memory/store.go

```diff
@@ -272,22 +231,7 @@ func (cr *store) Patch(orig config.Config, patchFn config.PatchFunc) (string, er
 		return "", errNotFound
 	}
 
-	rev := time.Now().String()
-	cfg.ResourceVersion = rev
 	ns.Store(cfg.Name, cfg)
 
-	return rev, nil
-}
-
-// hasConflict checks if the two resources have a conflict, which will block Update calls
-func hasConflict(existing, replacement config.Config) bool {
-	if replacement.ResourceVersion == "" {
-		// We don't care about resource version, so just always overwrite
-		return false
-	}
-	// We set a resource version but its not matched, it is a conflict
-	if replacement.ResourceVersion != existing.ResourceVersion {
```

Comment:
Set the resourceVersion to the current time when creating cfg. If resourceVersion is not empty it will always return true when executed in this place.